_stbt/logging.py

```diff
@@ -111,7 +111,7 @@ def append(self, **kwargs):
                 self.data[k] = []
             self.data[k].append(v)
 
-    def imwrite(self, name, image, region=None, colour=None):
+    def imwrite(self, name, image, regions=None, colours=None):
```

Comment:
I think regions and colours would be nicer as a list of tuples rather than two lists, but it's not important.